helpers.go

```diff
@@ -534,8 +534,22 @@ func GetRepoKeys(configs []PackageRepositoryConfig, cfg *RepoPlatformConfig, sOp
 				return nil, nil, err
 			}
 
-			keys = append(keys,
-				llb.AddMount(filepath.Join(cfg.GPGKeyRoot, name), gpgKey, llb.SourcePath(name)))
+			mountPath := filepath.Join(cfg.GPGKeyRoot, name)
+
+			opt := runOptionFunc(func(ei *llb.ExecInfo) {
+				keySt := gpgKey
+				inPath := filepath.Join("/tmp/in", name)
+				outPath := filepath.Join("/tmp/out", name)
+				keySt = worker.Run(
+					// dearmor key if necessary
+					ShArgs(fmt.Sprintf("cat '%s' | gpg --dearmor --output '%s'", inPath, outPath)),
```

Comment:
```suggestion
					ShArgs(fmt.Sprintf("gpg --dearmor --output '%s' < '%s'", inPath, outPath)),
```
This removes an extra call to `cat`